woocommerce/woocommerce-ios
# [Free Trial] Update login prologue screen

Closes: #9245 

# Why

This PR introduces the iOS counterpart of https://github.com/woocommerce/woocommerce-android/pull/8641 by updating the Login Prologue following the discussion in pe5pgL-2Ak#comment-2156

# Notes

I'm unsure if we need to update the WordPressAuthenticatorDisplayStrings properties following these text updates. I see that `enterYourSiteAddressButtonTitle` was directly related to the previous text, `Enter your store address`. 

However, it still triggers the Site address flow, so submitting a change to the `WordPressAuthenticator` code base just for this sounded unnecessary. Please correct me if my assumption is incorrect.

# How
Updates the Login Prologue view with three things:

- Updates the Prologue view title with a different font style and text
- Define the Prologue view subtitle with the new text
- Update the `Enter your store address` button text to `Log In`

# Testing Steps
- Fresh start the app and verify if the Login Prologue is displayed correctly
- Hit the `Log In` button and check if the Store address login flow starts
- Hit the `Get Started` button and check if the Account creation flow starts

# Screen Capture
| iPhone  | iPad |
| ------------- | ------------- |
| ![Simulator Screen Shot - iPhone 14 Pro - 2023-03-29 at 12 44 36](https://user-images.githubusercontent.com/5920403/228599515-7a912ade-5a0d-4e22-8dc7-94ef3875c6db.png) | ![Simulator Screen Shot - iPad Pro (12 9-inch) (6th generation) - 2023-03-29 at 12 46 45](https://user-images.githubusercontent.com/5920403/228599521-e21fb0e9-ce92-4ba3-8cd0-cf028d1353d1.png) |

### Android counterpart

<img src="https://user-images.githubusercontent.com/5920403/228600791-668e5686-1693-4f7e-b8ec-f5bbcf73147b.png" width="270" height="570" />

---
